docker/scripts/app-init.sh

```diff
@@ -54,17 +54,6 @@ else
     fi
 fi
 
-if [ ! -f "$WORKSPACEDIR/ietf/settings_local_sqlitetest.py" ]; then
-    echo "Setting up a default settings_local_sqlitetest.py ..."
-    cp $WORKSPACEDIR/docker/configs/settings_local_sqlitetest.py $WORKSPACEDIR/ietf/settings_local_sqlitetest.py
-else
-    echo "Using existing ietf/settings_local_sqlitetest.py file"
-    if ! cmp -s $WORKSPACEDIR/docker/configs/settings_local_sqlitetest.py $WORKSPACEDIR/ietf/settings_local_sqlitetest.py; then
-        echo "NOTE: Differences detected compared to docker/configs/settings_local_sqlitetest.py!"
-        echo "We'll assume you made these deliberately."
-    fi
-fi
-
```

Comment:
@NGPixel I don't think any devs used settings_local_sqlitetest. Did and actions? We don't have a comparable settings_local_postgrestest, and I suspect we don't need one?

Reply:
@rjsparks It is used in some workflows, e.g.:

https://github.com/ietf-tools/datatracker/blob/main/.github/workflows/build.yml#L163-L165

Reply:
But that's in the main branch, not feat/postgres. The corresponding lines are:

https://github.com/ietf-tools/datatracker/blob/feat/postgres/.github/workflows/build.yml#L156-L158

which use settings_postgrestest, not settings_local_sqlitetest